src/browser/provider/built-in/dedicated/chrome/cdp-client/index.ts

```diff
@@ -204,7 +204,7 @@ export class BrowserClient {
         this._runtimeInfo.emulatedDevicePixelRatio = this._config.scaleFactor || this._runtimeInfo.originalDevicePixelRatio;
     }
 
-    public async resizeWindow (newDimensions: Size): Promise<void> {
+    public async resizeWindow (newDimensions: { width: number, height: number}): Promise<void> {
```

Comment:
I believe it is needed because I changed the Size interface to also include outerHeight/outerWidth.
The resizeWindow method does not need those parameters and only needs width and height.

Reply:
fixed